### ISSUE TITLE
WCAG22 > quickref 1.4.10 Reflow Level AA

### ISSUE THREAD

**Issue by nileshprajapati** [w3c/wcag#4405]
**Quick reference**
https://www.w3.org/WAI/WCAG22/quickref/?showtechniques=1410#reflow

**Understanding**
https://www.w3.org/WAI/WCAG22/Understanding/reflow.html

The following items are missing from QuickRef compared to the understanding. 

**Sufficient Techniques**
- G224
- G225

**Advisory Techniques**
- Alternate view supporting Reflow for otherwise excepted content (Potential future technique)

**Comment by nileshprajapati** [comment]
https://www.w3.org/WAI/WCAG22/Techniques/general/G225

In G225, the word "Note" is displayed twice under Test > Procedure

**Comment by mbgower** [comment]
@kfranqueiro I have no idea how the quick ref material is formed. Do you?

**Comment by kfranqueiro** [comment]
Quickref has been republished today, with re-synchronized data from this repo. As a result, the discrepancies identified in this issue are now resolved.

The issue related to G225 has been tracked and resolved separately.
